src/MRubyD/Internals/InternalMRubyType.cs

```diff
@@ -0,0 +1,34 @@
+namespace MRubyD.Internals;
+
+internal enum InternalMRubyType:long
```

Comment:
Sorry for the delay in responding to this comment.
https://github.com/hadashiA/MRubyD/issues/7#issuecomment-2756861381

In this PR, they've separated InternalMRubyType and MRubyType, but I think it's fine to make the change to MRubyVType.
I don't think we need to worry about the compatibility of MRubyVType.